src/main/java/sdk/sample/Cleanup.java

```diff
@@ -0,0 +1,184 @@
+import sdk.sample.model.ModelCapacityPool;
+import sdk.sample.model.ModelNetAppAccount;
+import sdk.sample.model.ModelVolume;
+
+public class Cleanup {
+    /**
+     * Breaks and removes Data Replication connection and then deletes all resources -> volumes, pools and accounts
+     * @param config Project Configuration
+     * @param anfClient Azure NetApp Files Management Client
+     */
+    public static void runCleanup(ProjectConfiguration config, NetAppManagementClient anfClient)
+    {
+        if (config == null || config.getAccounts() == null)
+            return;
+
+        /*
+          Break and remove data replications
+         */
+        Utils.writeConsoleMessage("Breaking and removing Data Replication(s)...");
+        for (ModelNetAppAccount account : config.getAccounts())
+        {
+            if (account.getCapacityPools() != null)
+            {
+                for (ModelCapacityPool pool : account.getCapacityPools())
+                {
+                    if (pool.getVolumes() != null)
+                    {
+                        for (ModelVolume volume : pool.getVolumes())
+                        {
+                            if (volume.getSourceVolume() != null)
+                            {
+                                String[] params = {account.getResourceGroup(), account.getName(), pool.getName(), volume.getName()};
+                                VolumeInner destinationVolume = (VolumeInner) CommonSdk.getResource(anfClient, params, VolumeInner.class);
+                                if (destinationVolume != null)
+                                {
+                                    try
+                                    {
+                                        // Break replication on destination volume
+                                        anfClient.getVolumes().beginBreakReplication(account.getResourceGroup(), account.getName(), pool.getName(), volume.getName(), null).getFinalResult();
+                                        // Wait for replication status to be Broken
+                                        CommonSdk.waitForReplicationStatus(anfClient, account.getResourceGroup(), account.getName(), pool.getName(), volume.getName(), "Broken");
+                                        Utils.writeSuccessMessage("Successfully broke Volume Replication: " + destinationVolume.id());
+                                    }
+                                    catch (Exception e)
+                                    {
+                                        Utils.writeErrorMessage("An error occurred while breaking data replication: " + destinationVolume.id());
+                                        Utils.writeConsoleMessage("Error: " + e);
+                                        throw e;
+                                    }
+
+                                    try
+                                    {
+                                        Utils.threadSleep(30000);
```

Comment:
Why are you setting this thread sleep time ? the waitForReplicationStatus should do job